Multisig/UI/App/ContentView.swift

```diff
@@ -21,33 +23,43 @@ struct ContentView: View {
         // status bar (the navigation bar appears beneath the status bar
         // and it looks cropped) - this is seen on a real device (iPhone 6s)
         RootNavigationView {
-            TabView(selection: $selection){
-                AddSafeIntroView()
-                .tabItem {
-                    VStack {
-                        Image("tab-icon-balances")
-                        Text("Balances")
+            TabView(selection: $selection) {
+                assetsOrAddSafeIntroView
+                    .tabItem {
+                        VStack {
+                            Image("tab-icon-balances")
+                            Text("Assets")
+                        }
                     }
-                }
-                .tag(0)
+                    .tag(0)
 
                 AddSafeIntroView()
-                .tabItem {
-                    VStack {
-                        Image("tab-icon-transactions")
-                        Text("Transactions")
+                    .tabItem {
+                        VStack {
+                            Image("tab-icon-transactions")
+                            Text("Transactions")
+                        }
                     }
-                }
-                .tag(1)
+                    .tag(1)
 
                 SettingsView()
-                .tabItem {
-                    VStack {
-                        Image("tab-icon-settings")
-                        Text("Settings")
+                    .tabItem {
+                        VStack {
+                            Image("tab-icon-settings")
+                            Text("Settings")
+                        }
                     }
-                }
-                .tag(2)
+                    .tag(2)
+            }
+        }
+    }
+
+    var assetsOrAddSafeIntroView: some View {
```

Comment:
please extract it into a separate `AssetsContainerView`

Reply:
together with the fetch request

Reply:
it is good to encapsulate this and separate from the content view.

Reply:
done